config/crd/bases/cluster.x-k8s.io_clusters.yaml

```diff
@@ -892,7 +892,10 @@ spec:
                             instead of a custom JSON type, because controller-tools
                             has a hard-coded schema for apiextensionsv1.JSON which
                             cannot be produced by another type via controller-tools,
-                            i.e. it''s not possible to have no type field. Ref: https://github.com/kubernetes-sigs/controller-tools/blob/d0e03a142d0ecdd5491593e941ee1d6b5d91dba6/pkg/crd/known_types.go#L106-L111'
+                            i.e. it''s not possible to have no type field. Ref: https://github.com/kubernetes-sigs/controller-tools/blob/d0e03a142d0ecdd5491593e941ee1d6b5d91dba6/pkg/crd/known_types.go#L106-L111
```

Comment:
```suggestion
                            i.e. it's not possible to have no type field. Ref: https://github.com/kubernetes-sigs/controller-tools/blob/d0e03a142d0ecdd5491593e941ee1d6b5d91dba6/pkg/crd/known_types.go#L106-L111
```
nit

Reply:
Seems to be the way controller-tools generate it, by I worked around it